app/views/layouts/_nav_bar.html.erb

```diff
@@ -16,7 +16,7 @@
     <%= menu_item "Cart", showcart_path %>
     <%= menu_item "Admin Dashboard", admin_dashboard_path if current_user && current_user.admin? %>
     <% if current_user.nil? %>
-    	<%= menu_item "Signup", new_user_path %>
+    	<%= menu_item "Signup", choose_path %>
```

Comment:
This seems not RESTful...maybe there is some naming convention for switch methods? Choose_path just seems a little odd to me.